tasks/02_allegations/task.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python3
 
-from common import sql
+from common import db_config, sql
 
 import allegations as alleg
 
-from psycopg2.extras import DictCursor
+import sqlite3
```

Comment:
would it be preferable in any way to put the sqlite import into the if statement so that it only imports if necessary? i'm not sure what convention/best practices would suggest

Reply:
yeah, I think that's a good idea -- I'll make that update now!